README.md

```diff
@@ -161,15 +157,12 @@ See [reporting bugs][bug_guide] for details about reporting any issues.
 
 Operator Lifecycle Manager is under Apache 2.0 license. See the [LICENSE][license_file] file for details.
 
-[architecture]: /doc/design/architecture.md
-[philosophy]: /doc/design/philosophy.md
-[installation guide]: /doc/install/install.md
-[git_tool]:https://git-scm.com/downloads
-[go_tool]:https://golang.org/dl/
-[docker_tool]:https://docs.docker.com/install/
-[podman_tool]:https://github.com/containers/libpod/blob/master/install.md
-[buildah_tool]:https://github.com/containers/buildah/blob/master/install.md
-[kubectl_tool]:https://kubernetes.io/docs/tasks/tools/install-kubectl/
 [proposals_docs]: ./doc/contributors/design-proposals
 [license_file]:./LICENSE
 [bug_guide]:./doc/dev/reporting_bugs.md
+[olm-docs]: https://olm.operatorframework.io/
+[operator-framework-community]: https://github.com/operator-framework/community
+[operator-framework-communication]: https://github.com/operator-framework/community#get-involved
+[operator-framework-meetings]: https://github.com/operator-framework/community#meetings
+[contributor-documentation]: https://olm.operatorframework.io/docs/contribution-guidelines/
+[olm-getting-started]: https://olm.operatorframework.io/docs/getting-started/
```

Comment:
It looks like these changes are affecting the prerequisites section when I look at the preview markdown tab:

```
[git][git_tool]
[go][go_tool] version v1.12+.
[docker][docker_tool] version 17.03+.
Alternatively [podman][podman_tool] v1.2.0+ or [buildah][buildah_tool] v1.7+
[kubectl][kubectl_tool] version v1.11.3+.
Access to a Kubernetes v1.11.3+ cluster.
```

Reply:
solved